index.js

```diff
@@ -0,0 +1,51 @@
+import {
+  addButton,
+  titleInput,
+  authorInput,
+  sections,
+  navBar,
+  bookContainer,
+  DateAndTime,
+} from './dist/dom.js';
+
+import { BookCollection } from './dist/bookCollection.js';
+
+const { DateTime } = require('luxon');
+
+const bookCollection = new BookCollection([]);
+addButton.addEventListener(
+  'click',
+  bookCollection.addBook.bind(bookCollection, titleInput, authorInput),
+);
+
+navBar.addEventListener('click', (e) => {
+  if (e.target.classList.contains('nav-link')) {
+    const anchorHref = e.target.href.split('#')[1];
+    document.querySelectorAll('.section').forEach((section) => {
+      if (section.id === anchorHref) section.style.display = 'flex';
+      else section.style.display = 'none';
+    });
+  }
+});
+
+sections.forEach((section) => {
+  if (section === sections[1]) section.style.display = 'flex';
+  else section.style.display = 'none';
+});
+
+bookContainer.addEventListener('click', (e) => {
+  if (e.target.classList.contains('remove')) {
+    const removeBtn = e.target;
+    const book = bookCollection.bookArray.find(
+      (book) => book.id === removeBtn.id.split('-')[1],
+    );
+    bookCollection.removeBook(book);
+  }
```

Comment:
- [ ] You made the requested changes effectively :+1: , however, as pointed out by the previous reviewer, it seems that the remove function is not functioning correctly. It always removes the first element instead of the selected element. This might be happening because you are selecting elements using their class, and the function (`querySelector`) only takes the first element from the array of elements with that class.


https://user-images.githubusercontent.com/105117832/221854475-77f9c9d8-2f77-4432-b4fa-256f487495ed.mp4